lang/cs/Org.Apache.REEF.Tang/Implementations/Configuration/ConfigurationBuilderImpl.cs

```diff
@@ -85,6 +85,16 @@ protected ConfigurationBuilderImpl(IConfiguration[] confs) : this(new string[0],
         {
         }
 
+        public IConfigurationBuilder AddOptionalConfiguration(IConfiguration conf)
```

Comment:
Agree, deserves separate decision.
created JIRA to discuss: [REEF-1646] (https://issues.apache.org/jira/browse/REEF-1646)